data-processing-lib/python/src/data_processing/data_access/data_access_s3.py

```diff
@@ -68,32 +62,12 @@ def __init__(
         self.m_files = m_files
         self.n_samples = n_samples
         self.files_to_use = files_to_use
-
-        logger.debug(f"S3 access key provided: {access_key}")
-        logger.debug(f"S3 secret key provided: no soup for you!")
```

Comment:
why are you deleting all of this, it never hurts to have debugging logs.  and the get_*() methods are needed by spark.  would recommend only changing what is needed for this pr.

Reply:
get_methods are artificial, the do exactly the same that get from dict does. All the logs are printed by the factory already. Why have the code that is unnecessary

Reply:
IMO, the `get_*()` methods provide a nice encapsulation, hiding the internals from the module calling them, e.g. spark.  That being said, spark module would still work if it would use
```
data_access_s3.s3_credentials.get("access_key", None)
```
instead of:
```
data_access_s3.get_access_key()
```

Reply:
@cmadam Why does Spark need them in the first place? Is it using methods provided by the class or using the class as a placeholder for info?

Reply:
@blublinsky:  The local and S3 data access classes do not have the capability of reading a list of files into a Spark DataFrame; this operation is done using Spark methods. So Spark is only partially using the data access interface, to get a list of files to process, or to get the S3 credentials, but reading and writing to/from Spark DataFrames is being done outside the data access, as data access does not have those capabilities.

Reply:
@cmadam so which getters do you need? All of them?

Reply:
@cmadam the getters are back.